esm-samples/jsapi-custom-workers/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "dependencies": {
+    "@arcgis/core": "^4.19.0"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^19.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.0",
+    "css-loader": "^5.0.0",
+    "file-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.1",
+    "mini-css-extract-plugin": "^1.4.0",
+    "npm-run-all": "^4.1.5",
+    "rollup": "^2.51.1",
+    "rollup-plugin-terser": "^7.0.2",
+    "source-map-loader": "^2.0.1",
+    "webpack": "^5.28.0",
```

Comment:
Bump to latest versions: webpack current version is `5.39.0` and webpack-cli is `4.7.2`. I'm doing the same thing for the 4.20 update to the webpack sample.